index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7507e0edea.js" crossorigin="anonymous"></script>
+        <title>Personal portfolio</title>
+</head>
+<body>
+    <div>
+    <nav class="bar">
+       <a href="#"><h2 class="logo">My Logo</h2></a>
+        <div class="icon">
+            <a href="#" alt="bars"><i class="fa-solid fa-bars"></i></a>
+        </div>
+    </nav>
+</div>
+    <div>
+<header>
+    <h1 class="name">I’m JIPSON <br>
+        Glad to see you!</h1>
+  </header>
+        <p class="ABout"><b>I’m a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don’t hestiate to contact me.</b></p>
+        </div>
+            <br>
+                <h3>LET'S CONNECT</h3>
+            <div class="container"> 
+              <a href="#"><i class="fab fa-facebook-f"></i></a>
+              <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>  
+            </div>
```

Comment:
- [ ] It's more semantically correct if the social media icons were a list `ul` where each icon is a list item `li` that has an anchor tag `a` 👍 Kindly replace the div with the previously mentioned structure.
- [ ] Also, kindly try to follow the design provided in Figma for this section, the size, the color, and the spacings are much different.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192095585-7949a714-d482-4ac7-8992-03cba24f3f25.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192095600-0156baba-e2d7-4dab-8a46-b91d24c7c8ec.png)